dependencies.sh

```diff
@@ -8,10 +8,10 @@ export BYOND_MAJOR=514
 export BYOND_MINOR=1589
 
 #rust version
-export RUST_VERSION=1.54.0
+export RUST_VERSION=1.67.1
 
 #rust_g git tag
-export RUST_G_VERSION=0.4.7.1
```

Comment:
its considered v1.2.0 tag wise

Reply:
Oh, did the name and tag format backwards. I'll change it in the release to keep it consistent.